### ISSUE TITLE
Undeclared dependencies (Redis-Fast-0.17)

### ISSUE THREAD

**Issue by eserte** [shogo82148/Redis-Fast#55]
t/02-responses.t fails if Parallel::ForkManager or Test::UNIXSock are not installed. Probably these should be either declared or the test should be skipped if these are missing.

**Comment by shogo82148** [comment]
Parallel::ForkManager and Test::UNIXSock is required for test in Redis::Fast 0.17.
I declared these modules in cpanfile and META.json.

- https://github.com/shogo82148/Redis-Fast/blob/master/cpanfile#L16
- https://github.com/shogo82148/Redis-Fast/blob/master/META.json#L57

could you try to resolve dependencies again.

**Comment by eserte** [comment]
Problem seen on multiple cpantesters systems, not only on mine: http://matrix.cpantesters.org/?dist=Redis-Fast%200.17;reports=1

**Comment by eserte** [comment]
After running `Build.PL` manually (without `CPAN.pm`) I see the following in `MYMETA.json` — note the absence of all test prereqs:
```
   "prereqs" : {
      "build" : {
         "requires" : {
            "ExtUtils::CBuilder" : "0"
         }
      },
      "configure" : {
         "requires" : {
            "Module::Build" : "0.4005",
            "Module::Build::XSUtil" : "0.14"
         }
      }
   },
```

**Comment by shogo82148** [comment]
released Redis::Fast 0.18 and this issue is fixed.
